lib/block_orthogonalize.cu

```diff
@@ -246,12 +246,14 @@ namespace quda {
       } else if (Nvec == 24) {
         BlockOrthogonalize<vFloat, bFloat, nSpin, spinBlockSize, nColor, 24>(V, B, fine_to_coarse, coarse_to_fine,
                                                                              geo_bs, n_block_ortho);
+#ifdef GPU_WILSON_DIRAC
       } else if (Nvec == 32) {
```

Comment:
FYI---I've since replaced these `GPU_WILSON_DIRAC` and `GPU_STAGGERED_DIRAC`s with `NSPIN4` and `NSPIN1`, respectively.